plugins/offload_vio/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+find_package(eCAL)
```

Comment:
Missing `ecal`, `zed` and `glm` package.

Reply:
Can you clarify? The zed and eCAL dependencies are given on the build instructions. The glam is missing and I have added that one.